README.md

```diff
@@ -31,14 +31,24 @@ pip install -r <path_to_pyvcs>\requirements.txt
 From bash run:
 
 ```bash
-<path_to_pyvcs-python>/python.exe -m venv <path_to_virtual_env>
+<path_to_pyvcs-python>/python -m venv <path_to_virtual_env>
 source <path_to_virtual_env>/bin/activate
 pip install cython
 pip install --no-binary :all: --upgrade cffi==1.14.6 --no-use-pep517
 pip install --no-binary :all: -U numpy==1.21.2 --no-use-pep517
 pip install -r <path_to_pyvcs>/requirements.txt
 ```
+### Fedora
+
+From bash run:
 
+```bash
+<path_to_pyvcs-python>/python -m venv <path_to_virtual_env>
+pip install cython
```

Comment:
```suggestion
source <path_to_virtual_env>/bin/activate
pip install cython
```

Reply:
I didn't use the source command and PyVCS worked correctly by the way; I didn't include the source command because when I enter the "activate" source and I run the PIP command to install Cython it just throws errors to the terminal and I'm forced to make a new terminal window.

Reply:
Can you run `python` and confirm that it's picking up the correct executable? Without sourcing the virtual env I would expect it's just using your global version.

You should see something like this, specifically with the `pyvcs` part.

```
python
Python 3.9.6 (heads/pyvcs-dirty:7df244eee9, Jan  6 2022, 11:25:48) ...
Type "help", "copyright", "credits" or "license" for more information.
>>>```

Reply:
Nope. I see:
```
python
Python 3.10.1 (main, Dec  9 2021, 00:00:00) [GCC 11.2.1 20211203 (Red Hat 11.2.1-7)] on linux
Type "help", "copyright", "credits" or "license" for more information.
```
It runs fine even in normal python.

Reply:
Yeah, I've noticed it actually appears to run fine with normal python after some changes I made, although it's subtly running incorrectly.

I should probably try to detect this situation and raise an error. 

Normally, the system is supposed to be running in step with the user's code; the custom interpreter enforces this. But the user programs should use two api functions `wait_for_hsync` and `wait_for_vsync` to keep synchronized with the system. These two functions will catch up the system to the next horizontal or vertical checkpoint.

So what's happening when you run with the default python interpreter is that the user program runs for a while, then it calls one of those functions and then all of the system runs, instead of being interspersed. 

This should be a lot more clear once I write docs about the programming model 😅 

But, in short, sounds like you hit some errors getting the custom one to build/install.

Reply:
And also I think there is another problem: Your custom Python interpreter's name conflicts with the pre-installed python interpreter, so I think you should change its executable name to `pyvcs-python` instead of just `python`. This could easily be resolved by just renaming the executable, and also you should include an `install.sh` script like this:
```bash
sudo cp <pyvcs-python-path>/pyvcs-python /usr/bin
```
This is so that your custom python interpreter can be accessed globally from any directory.
I also think the best solution for detecting if the user is using the correct python interpreter is by checking the 0th argument (And having the executable name of the pyvcs python interpreter renamed from `python` to `pyvcs-python` or anything different from the normal python interpreter, to avoid conflict).

Reply:
That's a good suggestion; I didn't want to mess with the output of the compilation too much for fear of breaking something, but I'll try that out and make sure that it doesn't conflict; specifically when installing the extra requirements.

Were you able to resolve those errors and get it to run under pyvcs-python?

Reply:
I don't know; I saw it run how I thought it should have run, so I didn't try to run it on pyvcs-python.